mesh/accounts/models.py

```diff
@@ -1,14 +1,24 @@
+# in accounts folder: (accounts.models) models.py
 from django.db import models
 
-class Account(models.Model):
+class Accounts(models.Model):
```

Comment:
We recently found out that django admin adds an S to the table name already so the S becomes redundant.